Sources/JWTKeychain/Support/Mailer.swift

```diff
@@ -0,0 +1,67 @@
+import Vapor
+import SMTP
+import Transport
+
+public class Mailer {
```

Comment:
See my other comments related to this class. Basically I think we should introduce a SMTP-agnostic protocol so any consumer can send mails as they prefer. However, it would be nice to include this as a default option.